src/fmeval/eval_algorithms/qa_accuracy.py

```diff
@@ -116,6 +118,46 @@ def _f1_score(model_output: str, target_output: str, *, normalize_text: bool = F
         return float(ret)
 
 
+def _precision(model_output: str, target_output: str, *, normalize_text: bool = False) -> float:
+    """
+    Given the model output and the target output, compute the precision.
+    Precision is the fraction of words in the prediction that are also found in the target output.
+    Before computing precision, we normalize the text following the QuAC protocol.
```

Comment:
what's the QuAC protocol? Can you add a link?

Reply:
The function `_normalize_text_quac_protocol` in the same file [here](https://github.com/aws/fmeval/blob/30cc7c4d2692f19e7fe7927f2f60bcde9da96ccd/src/fmeval/eval_algorithms/qa_accuracy.py#L78) has the info.